ykrt/src/mt.rs

```diff
@@ -75,6 +82,23 @@ impl MT {
             .store(hot_threshold, Ordering::Relaxed);
     }
 
+    /// Return this `MT` instance's current trace failure threshold. Notice that this value can be
+    /// changed by other threads and is thus potentially stale as soon as it is read.
+    pub fn trace_failure_threshold(&self) -> TraceFailureThreshold {
+        self.inner.trace_failure_threshold.load(Ordering::Relaxed)
+    }
+
+    /// Set the threshold at which a `Location` from which tracing has failed multiple times is
+    /// marked as "do not try tracing again".
+    pub fn set_trace_failure_threshold(&self, trace_failure_threshold: TraceFailureThreshold) {
+        if trace_failure_threshold < 1 {
```

Comment:
Presumably a value of zero would effectively disable the JIT?

Reply:
Dunno -- I don't even want to think about what it does (or should...) mean!